Traceback (most recent call last):
  File "/home/orestas41/project-FootballPredict/lithuanian-football-league-predictor/training_validation/run.py", line 146, in <module>
    go(args)
  File "/home/orestas41/project-FootballPredict/lithuanian-football-league-predictor/training_validation/run.py", line 45, in go
    xgb_config['random_state'] = args.random_seed
AttributeError: 'Namespace' object has no attribute 'random_seed'